test/e2e/support/utils/is-wp-path.js

```diff
@@ -8,6 +8,13 @@ import { URL } from 'url';
  */
 import { getUrl } from './get-url';
 
+/**
+ * Checks if current url is a WordPress path.
+ *
+ * @param {string} WPPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
```

Comment:
`query` is optional so it should be `{?string}`.

Reply:
Adjusted, thx.